Design.md

```diff
@@ -276,3 +277,39 @@ There have been multiple rounds of CPU vulnerabilities (L1TF and MDS) which cann
 By default, Fedora CoreOS will configure the kernel to disable SMT on vulnerable machines. This conditional approach avoids incurring the performance cost on systems that aren't vulnerable. However, it fails to protect systems affected by undisclosed SMT vulnerabilities, and it allows future OS updates to disable SMT without notice if new vulnerabilities become known.
 
 We will document this policy and its consequences, and provide instructions for unconditionally enabling or disabling SMT for users who prefer a different policy.
+
+## Bucket Layout
+
+Originally discussed in [#189](https://github.com/coreos/fedora-coreos-tracker/issues/189).
+
+The `fcos-builds` bucket, fronted by http://builds.coreos.fedoraproject.org/ will be structured as follow:
+
+```
+/
+  prod/
+    streams/
+      stable/
+        stream.json
+        release-index.json
```

Comment:
This is [`releases.json`](https://github.com/coreos/mantle/pull/1013#issuecomment-505103522)

Reply:
If there is no technical reason, my preference is to keep filename release-index.json  as it avoids confusion while talking about release or release index file

Reply:
@sinnykumari I don't have a particularly strong preference either way. I was mostly noting that initially we defined it as `releases.json` and this would be a change to `release-index.json` which we would need to propogate through other tools / discussions / tickets.